src/constants/index.types.ts

```diff
@@ -0,0 +1,47 @@
+export interface FolderData {
+  name: string;
+  owner: {
+    profileImageSource: string;
+    name: string;
+  };
+  links: ObjectLink[];
+  count: number;
+}
+
+export interface FormetLink {
```

Comment:
오타인 것 같네요!

그리고 이렇게 특정 타입(여기서는 Link)을 기반으로 변형되는 타입들은, 원래 있던 타입을 재활용해서 정의해주면 좋습니다! Omit같은 유틸리티 타입이나, Indexed Access Types를 적용해보면 좋을 것 같아요!
```suggestion
export interface FormatLink {
```